change_log/next/FE-1455-dialog_FE-1648-sidebar.yml

```diff
@@ -0,0 +1,2 @@
+New Features: "Add trap-focus. It allows to keep focus inside of the element while tabbing. 
```

Comment:
Please remove this file and add the detail into the commit message. We no longer use renogen.

Reply:
done